src/lib/listeners/LogEventListener.ts

```diff
@@ -58,6 +59,7 @@ export class LogEventLogger implements LogEventListener {
 	}
 
 	public stopListening(): void {
+		this.logWriter.writeToLog('\n' + getMessage(BundleName.Shared, 'log.give-us-feedback'));
```

Comment:
Yeah that's a good question. I had the same thought. I'm not sure if the goal is to make the logs machine readable or not. If so - then we'd need to adjust the formatting to be more deterministic since currently messages can already go to multiple lines and possibly corrupt the formatting.

I currently don't think we need the log to be machine readable - my guess is that it will always just be for human readability when diagnosing issues. So I think having a newline plus a plain string works here.

Reply:
Fair enough.